README.md

```diff
@@ -467,27 +467,24 @@ form +++
                 $0.placeholder = "Tag Name"
             }
         }
-        $0 <<< NameRow() {
```

Comment:
This seems to come from the demo code, not explained in the readme, so removing to avoid confusion.